topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.md

```diff
@@ -0,0 +1,379 @@
+---
+layout: tutorial_hands_on
+title: Finding the muon stopping site with pymuon-suite in Galaxy 
+level: Introductory
+zenodo_link: https://zenodo.org/records/10219558
+questions:
+- "What are stopping sites and why is finding them important?"
+- "What computational techniques can be used to find stopping sites?"
+- "How can Galaxy help me with my analysis?"
+objectives:
+- "Understand why finding stopping sites is useful"
+- "Use pymuon-suite in Galaxy to find stopping sites"
+- "Understand the limitations of these tools and techniques"
+time_estimation: 1H
+key_points:
+- Galaxy supports multiple PyMuonSuite tools which can be used together to find muon stopping sites
+- The workflow for finding the stopping site involves three stages - structure generation, structure optimisation, and structure clustering
+- Within the UEP, the optimisation stage is based on analytically calculating the electrostatic forces applied to the muon, which can be performed via Galaxy.
+- The approach requires a CASTEP DFT calculation to be completed as a starting point
+contributions:
+  authorship:
+    - leandro-liborio 
+    - elichad
+    - patrick-austin
+  editing:
+    - leandro-liborio
+subtopic: muon-spectroscopy
+priority: 2
+abbreviations:
+  AIRSS: Ab Initio Random Structure Search
+  DFT: Density Functional Theory
+  UEP: Unperturbed Electrostatic Potential
+---
+
+
+# Introduction
+
+The problem of identifying the stopping site of the muon is often key in the
+interpretation of muon experimental results. As it was mentioned in the lecture, 
+the problem of finding the muon stopping site has been previously
+tackled using computational tools, and one of the most common methods is the so-called
+Unperturbed Electrostatic Potential (UEP) method, which identifies proposes that 
```

Comment:
```suggestion
{UEP} method, which identifies proposes that 
```

on first use of `{<abbreviation>}` it'll be expanded, if you want to help your learners remember and refresh their knowledge of this acronym easily later, just enclose more uses of `UEP` with braces, as `{UEP}`, and it'll automatically be turned into an HTML abbreviation. (you may wish to do this for other abbreviations you've defined as well.)

Reply:
Thanks for picking up on this - I really like the abbreviations feature, I think some of these may have been present originally but lost during editing and rewrites etc. I've put them back in where I could find them.